my-app/src/pages/Discover.js

```diff
@@ -1,12 +1,144 @@
+import React, { useState, useEffect } from 'react';
 import '../App.css';
 import './Discover.css'
+import { getAllUsers } from './DiscoverAPI.js'
```

Comment:
Doesnt seem like there's any routing being implemented using `react-router-dom`. I would suggest tying the pages together with this